backend/infrahub/webhook/models.py

```diff
@@ -54,6 +54,9 @@ def from_object(cls, obj: CoreWebhook) -> Self:
                 "infrahub.resource.label": f"!{registry.default_branch}",
             }
 
+        if obj.node_kind.value:
```

Comment:
I think we should only apply this match to the event that are compatible

Reply:
You mean that we'd reject it if "all" was chosen as the event_type?